### PR TITLE
Disabled local search

### PR DESCRIPTION
**Pull request by rkaron**
Per: https://jorsek.atlassian.net/browse/PT-3027
I'll document how to enable the local search: https://jorsek.atlassian.net/browse/ID-513

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -44,12 +44,12 @@ module.exports = {
...
   ],
   plugins: [
-    [require.resolve('@cmfcmf/docusaurus-search-local'), {
-        indexDocs: true,
-        docsRouteBasePath: '/',
-        indexDocSidebarParentCategories: 3,
-        indexPages: false,
-        language: "en"
-    }]
+    //[require.resolve('@cmfcmf/docusaurus-search-local'), {
```
Comment (KrystianBalwierzJorsek) [inline docusaurus.config.js]: please add here a description of why it is commented out

File: package.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
     "serve": "docusaurus serve"
   },
   "dependencies": {
-    "@cmfcmf/docusaurus-search-local": "latest",
```
Comment (KrystianBalwierzJorsek) [inline package.json:14]: should it be commented-out or removed?

Comment (rkaron) [reply package.json:14]: JSON doesn't like comments.
image

Comment (KrystianBalwierzJorsek) [reply package.json:14]: ah, right, so it should be described only in docs what should be done to enable it

File: docusaurus.config.js (hunk 1 above)
Comment (rkaron) [reply docusaurus.config.js]: Fixed in commit: 6190797accdad166b10e8d510999e070fd34edae

**Review by KrystianBalwierzJorsek — APPROVED** [review]

**Review by KrystianBalwierzJorsek — APPROVED** [review]

**Pull request merged**